scripts/build_scripts/build_with_profiler_opt.sh

```diff
@@ -11,7 +11,7 @@ if [[ -z "$ARCH_NAME" ]]; then
     exit 1
 fi
 
-ENABLE_TRACY=1 ENABLE_PROFILER=1 cmake -B build -G Ninja -DCMAKE_CXX_COMPILER=clang++-17
```

Comment:
Actually, why did we need this in the first place? Wasn't clang the default?

Reply:
No we didn't force Clang-17 back then, but now we have to.